src/Core/Commander/Providers/BuildingBox_Provider.js

```diff
@@ -19,7 +19,7 @@ import WFS_Provider from 'Core/Commander/Providers/WFS_Provider';
 import * as THREE from 'three';
 import Ellipsoid from 'Core/Math/Ellipsoid';
 import GeoCoordinate,{UNIT} from 'Core/Geographic/GeoCoordinate';
-import CVML from 'Core/Math/CVML';
+import Earcut from 'Earcut';
```

Comment:
error in browser
`Module not found: Error: Cannot resolve module 'Earcut'`
replace
```js
import Earcut from 'Earcut';
```

by

```js
import Earcut from 'earcut';
```